src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,30 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new InvalidUserException("Incorrect value");
```

Comment:
The exception message 'Incorrect value' is not informative enough. It should clearly state what is incorrect, for example, 'User cannot be null'.